notebooks/demo-notebook.Rmd

```diff
@@ -249,12 +250,12 @@ The final property of interest is **`measuredValue`**. This property tells us th
 
 These parameters are useful for deciding what values to provide to the API. To query for `Observation`s using the R API Client, we call **`GetObservations`** which accepts the following parameters.
 
-- **`dcids`** - A vector or single column tibble/data frame of dcids of nodes that are observed by observations being queried for.
-- **`measuredProperty`** - The `measuredProperty` of the observation.
-- **`statsType`** - The statistical type of the observation.
-- **`observationDate`** - The `observationDate` of the observation.
-- **`observationPeriod`**`[=NULL]` - An optional parameter specifying the `observationPeriod` of the observation.
-- **`measurementMethod`**`[=NULL]` - An optional parameter specifying the `measurementMethod` of the observation.
+- **`dcids`** - vector OR single column tibble/data frame of dcids to get observations for.
+- **`measuredProperty`** - string identifying the measured property.
+- **`statsType`** - string identifying the statistical type for the observation.
+- **`observationDate`** - string of the associated observation date in ISO8601 format.
+- **`observationPeriod`** - optional string specifying the observation period. NULL by default.
```

Comment:
These are not truly optional in the sense that, if they are specified in the observation, they must be specified.

Reply:
added this in f18ffa